src/plusplus.js

```diff
@@ -43,31 +43,25 @@ const _ = require('underscore');
 const clark = require('clark');
 const querystring = require('querystring');
 const ScoreKeeper = require('./scorekeeper');
+const { WebClient } = require('@slack/client');
```

Comment:
I suppose we are just accepting this will only work with Slack, no fallback to other hubot adapters?

Reply:
patches accepted :)